contracts/mixnet/src/msg.rs

```diff
@@ -0,0 +1,24 @@
+use cosmwasm_std::HumanAddr;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+use crate::state::MixNode;
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+pub struct InitMsg {}
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
+#[serde(rename_all = "snake_case")]
+pub enum HandleMsg {
+    RegisterMixnode { mix_node: MixNode },
+    UnRegisterMixnode {},
```

Comment:
I guess it might be slightly pedantic, but isn't "unregister" a single word, i.e. it should be `UnregisterMixnode`?